### PR TITLE
✨ [hub/collections] document the image gallery/viewer

### PR DESCRIPTION
**Pull request by SBrandeis**
# TL;DR

New section under the collections documentation showcasing the image gallery feature:

https://moon-ci-docs.huggingface.co/docs/hub/pr_1032/en/collections#adding-images-to-a-collection-item

image

### CODE REVIEW

**Review by Kakulukian — APPROVED** [review]
👌🏻

**Review by julien-c — COMMENTED** [review]
lgtm cc @osanseviero

File: docs/hub/collections.md [hunk 1]
```diff
@@ -75,6 +75,24 @@ It's possible to add a note to any item in a collection to give it more context
...
+
+Similarily, you can attach images to a collection item. This is useful for showcasing the output of a model, the content of a dataset, attaching an infographic for context, etc.
+
+To start adding images to your collection, you can click on the image icon in the contextual menu of an item. The menu shows up when you hover over an item with your mouse.
+
+![Collection image icon](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/collections-image-button.webp)
+
+Then, add images by dragging and dropping images from your computer. You can also click on the gray zone to select image files from your computer's file system.
+
+![Collection image drop zone](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/collections-image-dropzone.webp)
```
Comment (gary149) [inline docs/hub/collections.md:88]: maybe you can remove this one? to limit the nb of images in the docs.

**Review by gary149 — APPROVED** [review]

File: docs/hub/collections.md (hunk 1 above)
Comment (SBrandeis) [inline docs/hub/collections.md:88]:
```suggestion
```

File: docs/hub/collections.md [hunk 2]
```diff
@@ -75,6 +75,23 @@ It's possible to add a note to any item in a collection to give it more context
 
 ![Collection note](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/collection-note.webp)
 
+### Adding images to a collection item
+
+Similarily, you can attach images to a collection item. This is useful for showcasing the output of a model, the content of a dataset, attaching an infographic for context, etc.
+
+To start adding images to your collection, you can click on the image icon in the contextual menu of an item. The menu shows up when you hover over an item with your mouse.
+
+![Collection image icon](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/collections-image-button.webp)
```
Comment (osanseviero) [inline docs/hub/collections.md:84]: Please also add dark mode images

**Review by osanseviero — APPROVED** [review]
Very nice! I just used this in this collection! https://huggingface.co/collections/osanseviero/os-week-highlights-oct-9-15-6525053d02387d10b432145d

File: docs/hub/collections.md (hunk 2 above)
Comment (SBrandeis) [reply docs/hub/collections.md:84]: I think keeping only images in light theme is on purpose cc @gary149

Comment (osanseviero) [reply docs/hub/collections.md:84]: In the rest of the docs we have images in dark mode as well

Comment (gary149) [reply docs/hub/collections.md:84]: I don't think so in most places, but yes, but yes it would be nicer.

Comment (SBrandeis) [reply docs/hub/collections.md:84]: Opened https://github.com/huggingface/hub-docs/issues/1037

**Pull request merged**